database/plugin/metadata/sqlite/database.go

```diff
@@ -131,9 +133,56 @@ func (d *MetadataStoreSqlite) init() error {
 	if err := d.db.Use(tracing.NewPlugin(tracing.WithoutMetrics())); err != nil {
 		return err
 	}
+	// Schedule daily database vacuum to free unused space
+	d.scheduleDailyVacuum()
 	return nil
 }
 
+func (d *MetadataStoreSqlite) runVacuum() error {
+	if d.dataDir == "" {
+		return nil
+	}
+	// Open a new connection to the DB only for vacuuma
```

Comment:
Spelling of "vacuum"

Reply:
Can you tell I added a line after that one 😅